test/deflate.js

```diff
@@ -0,0 +1,5 @@
+const pako = require("pako");
+
+const compressed = pako.deflateRaw(new Uint8Array(Buffer.from(process.argv[2], 'hex')), { level: 9 });
+
+console.log(Buffer.from(compressed).toString('hex'));
```

Comment:
🚀 

cc @nalinbhardwaj this is a good example of why we might not want to require `IInflator` or `IOpInflator` to come with a `compress()` function in solidity/evm

in this case, there's an existing library that *decompresses* a `DEFLATE` stream, but none that compresses > easier to compress in js